to/src/to/to_trait.rs

```diff
@@ -1,12 +1,45 @@
+//! TODO: Mod docu
```

Comment:
Is adding mod documentation out of the scope for this PR?

Reply:
Should be fixed in https://github.com/rosenpass/rosenpass/pull/556

Reply:
Lets see if that CI passes, then @DavidNiehues should cherry pick my fixes into this PR.